packages/dashboard/src/components/Card/GridList/GridList.tsx

```diff
@@ -44,6 +48,17 @@ export class GridList extends React.Component<IGridListProps> {
     );
   }
 
+  private _onRenderRow(props: IDetailsRowProps): JSX.Element {
+    const styles: Partial<IDetailsRowStyles> = {
+      root: {
+        borderBottom: 'unset'
```

Comment:
>unset [](start = 23, length = 5)

Did you test this in IE. I am not sure unset in supported in IE

Reply:
https://caniuse.com/#feat=css-unset-value IE 11 does not seem to support unset. borderBottom: 'none' should work instead.